from unittest import TestCase


class TestCall(TestCase):
    def test_init_from_file(self):
        self.fail()
    pass
